src/librustc_errors/emitter.rs

```diff
@@ -1013,8 +1013,19 @@ impl EmitterWriter {
 
                 // Then, the secondary file indicator
                 buffer.prepend(buffer_msg_line_offset + 1, "::: ", Style::LineNumber);
+                let loc = if let Some(first_line) = annotated_file.lines.first() {
+                    let col = if let Some(first_annotation) = first_line.annotations.first() {
+                        format!(":{}", first_annotation.start_col + 1)
+                    } else { "".to_string() };
```

Comment:
Can you leave the empty string on it's own line? It is easier to skip the code if the blocks are consistent.

Reply:
Yep, will do!